tkg/client/update_credentials.go

```diff
@@ -197,3 +212,120 @@ func (c *TkgClient) updateVSphereCredentialsForCluster(clusterClient clusterclie
 
 	return nil
 }
+
+// UpdateAzureClusterCredentials update azure cluster credentials
+func (c *TkgClient) UpdateAzureClusterCredentials(clusterClient clusterclient.Client, options *UpdateCredentialsOptions) error {
+	if options.AzureUpdateClusterOptions.AzureTenantID == "" || options.AzureUpdateClusterOptions.AzureClientID == "" || options.AzureUpdateClusterOptions.AzureClientSecret == "" {
+		return errors.New("either tenantId, clientId or clientSecret should not be empty")
+	}
+
+	if !options.IsRegionalCluster {
+		unifiedIdentity, err := clusterClient.CheckUnifiedAzureClusterIdentity(options.ClusterName, options.Namespace)
+		if err != nil {
+			return err
+		}
+		if unifiedIdentity {
+			log.Warningf("AzureCluster %s use the same AzureClusterIdentity with Management Cluster. It cannot be updated separately.", options.ClusterName)
```

Comment:
How about rephrase as: "... uses the same AzureClusterIdentity from its management cluster. ..."

Reply:
Done.